starter_code/components.js

```diff
@@ -0,0 +1,31 @@
+
+
+class Image extends React.Component {
+  constructor(){
+    super();
+    this.state = {showResults: true}
+  }
+  _toggleImage(event){
+    this.setState( {showResults: !this.state.showResults} );
```

Comment:
Good use of the `!` operator to keep this as a short one-liner instead of a longer conditional.